freqtrade/freqtradebot.py

```diff
@@ -268,11 +268,19 @@ def get_free_open_trades(self) -> int:
 
     def update_funding_fees(self):
         if self.trading_mode == TradingMode.FUTURES:
-            for trade in Trade.get_open_trades():
-                funding_fees = self.exchange.get_funding_fees_from_exchange(
-                    trade.pair,
-                    trade.open_date
-                )
+            trades = Trade.get_open_trades()
+            for trade in trades:
+                if self.config['dry_run']:
+                    funding_fees = self.exchange.calculate_funding_fees(
```

Comment:
can we move this check into `get_funding_fees()` within the exchange class?

the distinction (dry/non-dry) is done within the exchange wherever possible - and i don't see any benefits to having this here.